sessionctx/variable/tidb_vars.go

```diff
@@ -528,6 +532,7 @@ const (
 	DefTiDBHashAggPartialConcurrency   = ConcurrencyUnset
 	DefTiDBHashAggFinalConcurrency     = ConcurrencyUnset
 	DefTiDBWindowConcurrency           = ConcurrencyUnset
+	DefTiDBStreamAggConcurrency        = 1
```

Comment:
at the moment, this feature is not stable, so we set it to 1, which means it's not enabled by default, this will be changed to `ConcurrencyUnset` in the future.